MyQueue.java

```diff
@@ -0,0 +1,64 @@
+// Time Complexity : Array operations, no iterations so O(1)
+// Space Complexity : Two stacks so twice O(N) ~ O(N)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Yes, some constraints doubts like, what default values to be set for peek and pop if they are empty. 
```

Comment:
> for peek and pop if they are empty.

This will be specified in the question. If in an interview, this will be a clarifying question that should be asked.